### DISCUSSION TITLE
Move me to an Emeritus Maintainer position

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#309] [General]
Hi team, I am pursuing another opportunity, and will not have as much time to dedicate to maintaining the project in the future. I would like to request to be moved into an Emeritus Maintainer position for the subteams I maintain. It has been a pleasure to be a part of Paketo for the last few years.
This includes:
- .NET Core @paketo-buildpacks/dotnet-core-maintainers 
- Ruby @paketo-buildpacks/ruby-maintainers 
- PHP @paketo-buildpacks/php-maintainers
- Tooling @paketo-buildpacks/tooling-maintainers 
- Dependencies @paketo-buildpacks/dependencies-maintainers 
- Stacks @paketo-buildpacks/stacks-maintainers 

@paketo-buildpacks/steering-committee please let me know if you're able to make this change? Thank you!

**Comment by dmikusa** [comment]
- Removed from teams
- https://github.com/paketo-buildpacks/community/pull/168

**Comment by loewenstein-sap** [comment]
I just realized that @dmikusa's #168 seemed to only move @sophiewigmore from maintainer to emeritus. What about the contributor roles? Should we move that as well or do you plan to further contribute to the project, just with far less time commitment?

**Reply by dmikusa** [reply]
I asked @sophiewigmore and she requested to stay on for the contributor roles so I kept those.

**Reply by loewenstein-sap** [reply]
This is great to hear.
